lib/Backend/SccLiveness.cpp

```diff
@@ -379,7 +379,7 @@ SCCLiveness::ProcessSrc(IR::Opnd *src, IR::Instr *instr)
 		IR::SymOpnd *symOpnd = src->AsSymOpnd();
 		RegNum reg = LinearScanMD::GetParamReg(symOpnd, this->func);
 
-		if (reg != RegNOREG && !PHASE_OFF(Js::LivenessPhase, this->func))
+		if (reg != RegNOREG && PHASE_ON(Js::RegParamsPhase, this->func))
```

Comment:
>		 [](start = 0, length = 2)

nit: seems tab is used in this new code block (else if block) instead of 4 spaces.